IdentityCore/src/cache/serializers/MSIDCacheItemJsonSerializer.m

```diff
@@ -78,16 +70,7 @@ - (NSData *)serializeAccountCacheItem:(MSIDAccountCacheItem *)item
 
 - (MSIDAccountCacheItem *)deserializeAccountCacheItem:(NSData *)data
```

Comment:
nit: This one and other similar methods are not used anymore.

Reply:
Yes! Good catch :)